.env

```diff
@@ -0,0 +1 @@
+DB_CONNECTION = mongodb+srv://BeyFaith:<Fr_90210>@cluster0.2xoap.mongodb.net/<Shopping-Cart>?retryWrites=true&w=majority
```

Comment:
I see your env variable are pushed on github. you might want to remove them

Reply:
Hey,Ijust saw this now,Removing it now